README.md

```diff
@@ -91,11 +91,11 @@ To run the backend tests, navigate to the `backend/` directory and run the follo
 yarn test
 ```
 
-This will execute all the tests located in the `__tests__/` folder.
+This will execute all the tests located in the `tests/` folder.
 
 **Writing Tests:**
 
```

Comment:
what does this mean?

Reply:
This means cat is love and cat will save the world. 🫶😺🫶